### PR TITLE
Improve SEO

### PR DESCRIPTION
**Pull request by MiguelHigueraDev**
I added a lot of SEO tags to improve its SEO ranking (duh) and also improve its look when linking it in platforms like Discord or Twitter. 

Now it should display more information and an image instead of just the URL.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,6 +1,24 @@
 <!DOCTYPE html>
 <html>
     <head>
+        <!-- Begin SEO tags-->
+        <meta name="description" content="Hi I'm Lemonbot and it's nice to meet you! I am a Discord bot with a lot of features like moderation, games, fun, etc." />
+		<meta name="keywords" content="discord, bot, fun, games, memes, moderation" />
+		<meta name="og:image" content="https://lemonbot.net/assets/lemonBot.png" />
```
Comment (iSlammedMyKindle) [inline index.html:7]: Do URLs in the meta tag require a full path? For example would simply `assets/lemonBot.png` work?

**Comment by iSlammedMyKindle** [comment]
I see there's a new PNG being added; that makes it sound like the meta tag would require PNGs and not vectors. Is that true?

File: index.html [hunk 2]
```diff
@@ -1,6 +1,24 @@
 <!DOCTYPE html>
 <html>
     <head>
+        <!-- Begin SEO tags-->
+        <meta name="description" content="Hi I'm Lemonbot and it's nice to meet you! I am a Discord bot with a lot of features like moderation, games, fun, etc." />
```
Comment (iSlammedMyKindle) [inline index.html:5]: I was looking around to see if this was something I wrote, but nothing shows up - This description is beautiful! it keeps the light-hearted tone of the website very nicely! Compliments to the chef!

File: index.html (hunk 1 above)
Comment (MiguelHigueraDev) [reply index.html:7]: Yes they require a full path to work.

File: index.html (hunk 2 above)
Comment (MiguelHigueraDev) [reply index.html:5]: Thanks! Yeah I wrote it myself based on what I read

**Comment by MiguelHigueraDev** [comment]
> I see there's a new PNG being added; that makes it sound like the meta tag would require PNGs and not vectors. Is that true?

Yes, that's why I added a PNG. Open Graph only supports PNGs and JPGs

**Comment by iSlammedMyKindle** [comment]
Thank's for the PR @MiguelHigueraDev ! I'll go ahead and merge this PR, the metadata is appreciated.

Once merged, github pages may take up to 15-30 minutes to deploy the code

**Pull request merged**